scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt

```diff
@@ -157,6 +157,8 @@ class RegexPatternValidationCheckTest {
   private val wikiReferenceNote =
     "Refer to https://github.com/oppia/oppia-android/wiki/Static-Analysis-Checks" +
       "#regexpatternvalidation-check for more details on how to fix this."
+  private val doesNotContainScreenName = "Screen Name not added."
```

Comment:
Nit: move this & the other above the note, and rename to match the other error message field names.

Reply:
done.